pkg/function/scale_test.go

```diff
@@ -269,7 +269,7 @@ func (s *ScaleSuite) TestGetArgs(c *C) {
 				},
 			},
 			args: map[string]interface{}{
-				ScaleWorkloadReplicas: 2,
+				ScaleWorkloadReplicas: "2",
```

Comment:
Let's leave one of these an int to test both codepaths

Reply:
@tdmanv we do have two of these as int.